lib/clientsidescripts.js

```diff
@@ -47,24 +47,46 @@ function wrapWithHelpers(fun) {
  *     be passed as a parameter.
  */
 functions.waitForAngular = function(rootSelector, callback) {
-  var el = document.querySelector(rootSelector);
-
-  try {
-    if (!window.angular) {
-      throw new Error('angular could not be found on the window');
+  if (!rootSelector) {
```

Comment:
Maybe I'm missing something, but wouldn't all ng1 tests fail right now? I guess by default the rootSelector is 'body' or something, but someone could have easily accidentally called 'useAllAngularAppRoots' for ng1. Maybe a safeguard?